model_analyzer/analyzer.py

```diff
@@ -123,7 +126,10 @@ def profile(self, client: TritonClient, gpus: List[GPUDevice], mode: str,
         if not self._config.skip_summary_reports:
             self._create_summary_tables(verbose)
             self._create_summary_reports(mode)
-            logger.info(self._get_report_command_help_string())
+
+            # TODO-TMA-650: Detailed reporting not supported for multi-model
```

Comment:
If someone runs A and B non-concurrently, do we suggest a report command that is now illegal?

Reply:
This does not (and has never) worked correctly for sequential profiling. Please review the code changes for this. The new output is:
```
16:18:57 [Model Analyzer] To generate detailed reports for the 3 best simple_int8 configurations, run `model-analyzer report --report-model-configs simple_int8_config_7,simple_int8_config_9,simple_int8_config_4 --export-path /swdev/model_analyzer --checkpoint-directory /swdev/profile_checkpoints/`
16:18:58 [Model Analyzer] To generate detailed reports for the 3 best add_sub configurations, run `model-analyzer report --report-model-configs add_sub_config_4,add_sub_config_6,add_sub_config_7 --export-path /swdev/model_analyzer --checkpoint-directory /swdev/profile_checkpoints/`
```